File: POO/Uber/js/UberVan.js
const Car = require('./Car.js');

class UberVan extends Car {

  constructor(licence, driver, typeCarAccepted, seatsMaterial) {
    super(licence, driver)
    this.typeCarAccepted = typeCarAccepted;
    this.seatsMaterial = seatsMaterial;
  }

}

module.exports = UberVan;

File: POO/Uber/python/uberVan.py
from car import Car

class UberVan(Car):
  type_car_accepted = []
  seats_material = []

  def __init__(self, licence, driver, type_car_accepted, seats_material):
    super().__init__(licence, driver)
    self.type_car_accepted = type_car_accepted
    self.seats_material = seats_material